### PR TITLE
First commit

### PR DESCRIPTION
**Pull request by romanbobak**

### CODE REVIEW

**Review by az4ert85 — COMMENTED** [review]
All is good. Accepted

File: index.html [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<html>
+    <body>
+        Hello world!
```
Comment (az4ert85) [inline index.html]: Please make text bold

**Review by az4ert85 — CHANGES_REQUESTED** [review]
Add head section

**Comment by romanbobak** [comment]
> Add head section

Added head section

**Review by az4ert85 — APPROVED** [review]

**Pull request merged**